challengeTwo/objcts.js

```diff
@@ -0,0 +1,10 @@
+var k = 0;
+var j;
+let numbers ={
+k : j
+}
+for(k in numbers){
+  j = k*k;
+    k++
+}
+console.log(Object.values(numbers))
```

Comment:
Always add newlines at the end of your files

Reply:
okay